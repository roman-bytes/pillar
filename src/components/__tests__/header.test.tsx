import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import Header from '../header';

describe('Header', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Header siteTitle="Starter" />).toJSON();

        expect(tree).toMatchSnapshot();
    });

    it('contains a link to the home page', () => {
        const { getByText } = render(<Header siteTitle="Starter" />);

        expect(getByText('Starter')).toHaveAttribute('href', '/');
    });
});
