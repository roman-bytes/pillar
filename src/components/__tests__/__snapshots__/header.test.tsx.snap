// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="bg-purple-800 mb-6"
>
  <div
    className="mx-auto max-w-4xl py-6 px-4"
  >
    <h1
      className="text-4xl"
    >
      <a
        className="text-white"
        href="/"
      >
        Starter
      </a>
    </h1>
  </div>
</header>
`;
